src/python/pants/jvm/resolve/coursier_fetch.py

```diff
@@ -111,37 +114,26 @@ def to_json_dict(self) -> dict:
             "artifact": self.artifact,
             "version": self.version,
             "packaging": self.packaging,
+            "classifier": self.classifier,
         }
         return ret
 
     @classmethod
-    def from_coord_str(cls, s: str, *, with_2315_workaround=False) -> Coordinate:
-        """Parses from a coordinate string with an optional `packaging` string.
+    def from_coord_str(cls, s: str) -> Coordinate:
+        """Parses from a coordinate string with optional `packaging` and `classifier` coordinates.
```

Comment:
Could you include a link to one of the implementations you referenced here?

Reply:
added link to Aether's implementation